RevenueCat/Scripts/Purchases.cs

```diff
@@ -41,9 +42,21 @@ public partial class Purchases : MonoBehaviour
     /// </summary>
     public delegate void GetPaymentDiscountFunc(PaymentDiscount paymentDiscount, Error error);
 
-    [Tooltip("Your RevenueCat API Key. Get from https://app.revenuecat.com/")]
+    [FormerlySerializedAs("revenueCatAPIKey")]
+    [ObsoleteAttribute("This property is obsolete. Use revenueCatAPIKeyIOS and revenueCatAPIKeyAndroid instead.", false)]
+    [Tooltip("(DEPRECATED) RevenueCat API Key. Get from https://app.revenuecat.com/. " +
```

Comment:
might be good to add that legacy keys will continue working

Reply:
good call